src/message_interface/mod.rs

```diff
@@ -107,12 +107,6 @@ where
     handle.listen(events, handler).await;
 }
 
-#[cfg(feature = "events")]
-/// Remove wallet event listeners, empty vec will remove all listeners
-pub async fn clear_listeners(handle: &WalletMessageHandler, events: Vec<WalletEventType>) {
```

Comment:
Add this to the Rust changelog please. Maybe merge develop as I just released a version so we need a new section for the future release.

Reply:
Done